tests/test_utils.py

```diff
@@ -176,3 +177,15 @@ def test_find_src_dir_errors_when_no_init_is_found(self, tmp_path: pathlib.Path)
             f"Is there an __init__.py file in your module?",
         ):
             _find_src_dir(output_folder)
+
+    def test_classify_with_threshold_works_on_classification_output(
+        self, classifier, train_iris_dataset
+    ):
+        data = train_iris_dataset.load_prediction_data(0)
+
+        pred = _classify(data, classifier.estimator)
+        assert pred == np.array([0])
+
+        pred = _classify(data, classifier.estimator, threshold=0.20)
+        # this should be true since the true proba exceeds the threshold?
```

Comment:
We need to pick the true classification when it exceeds the threshold, right? Right now the false is result if both exceeds the threshold

Reply:
Hmm, you're right - I think in this round we should focus on getting the binary situation handled - I'm not sure thresholding makes sense for multiple classes...